src/components/EmojiPicker/EmojiPickerMenu/index.tsx

```diff
@@ -114,29 +106,29 @@ function EmojiPickerMenu({forwardedRef, onEmojiSelected, activeEmoji}) {
         isActive: true,
     });
 
-    const filterEmojis = _.throttle((searchTerm) => {
+    const filterEmojis = lodashDebounce((searchTerm: string) => {
```

Comment:
Why was this changed to debounce? [Here](https://css-tricks.com/debouncing-throttling-explained-examples/) is a nice article to read about differences